### ISSUE TITLE
Model size hyperparameter

### ISSUE THREAD

**Issue by Wangweiyao** [Pointcept/PointTransformerV3#18]
Hi thanks for the great work!

I wonder do you have any suggestion on how to increase the model size? For example, I found that increase spconv kernel improve performance on our task. Do you see similar things? 

Thanks!

**Comment by Charlie839242** [comment]
I found that increasing the spconv kernel size causes series overfitting. Did you encounter similar situations?

**Comment by Gofinge** [comment]
> I found that increasing the spconv kernel size causes series overfitting. Did you encounter similar situations?

Exactly, SpConv is not good at expanding the receptive field. It is also the reason why we need attention. No need to scale up the kernel size of conv in PTv3.

**Comment by Gofinge** [comment]
> Hi thanks for the great work!
> 
> I wonder do you have any suggestion on how to increase the model size? For example, I found that increase spconv kernel improve performance on our task. Do you see similar things?
> 
> Thanks!

Hi, I highly recommend reading the paper, _Scaling Laws for Neural Language Models_, from OpenAI. So there is no need to force scaling up parameters. The scaling up of parameters always goes with the scaling up of data. If you have already scaled up the train data (scale), you can consider increasing the number of blocks and channels of PTv3. For the current training setting (even multi-dataset joint training from PPT), the model is still over capacity.
